forloop_modules/function_handlers/input_waiter.py

```diff
@@ -0,0 +1,26 @@
+# input_waiter.py
+from forloop_modules.globals.variable_handler import LocalVariable, variable_handler
+from forloop_modules.queries.node_context_requests_backend import post_console_log
+import forloop_modules.flog as flog
+
+_pending: dict[str, str] = {}
+_prompts: dict[str, str] = {} 
+
+def register_pending_input(node_uid: str, output_variable: str, prompt_text: str = None):
```

Comment:
The function is not enough human-understandable -> What is "register_pending_input" -> maybe some better name? Or Comment which describes what is this